drivers/wifi/nrf700x/Kconfig

```diff
@@ -401,3 +401,14 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config WIFI_NRF700X_SKIP_LOCAL_MAC
```

Comment:
Same comment as https://github.com/nrfconnect/sdk-nrfxlib/pull/1128/files#r1402077522 and even in the help text the focus should be on `co-located APs do not improve location` than checking for `locally administrative` bit in the MAC address.

Reply:
Updated the config name.